docs/DeveloperGuide.md

```diff
@@ -260,7 +260,7 @@ The above mentioned Parser class inherits the `#parse` method from the Parser in
 
 Subsequently, the created `ClearAssigneeCommand` object contains an `#execute` method which is responsible for
 clearing all assignees of the Task, with respect to its index. This is achieved by creating a new 
-`Task` object with the same fields and values as before but setting the `Set<Assignee>` field as a new empty HashSet. 
+`Task` object with the same fields and values as before but with the `Assignee` field set to be a new empty HashSet. 
```

Comment:
i think it should be assignee field cause you are referring to the variable name not the class